vcluster/configure/vcluster-yaml/policies/network-policy.mdx

```diff
@@ -44,24 +44,50 @@ policies:
     enabled: true
 ```
 
-#### Custom egress rules {#custom-egress}
-
-Control outbound traffic with specific CIDR blocks:
-
+#### Custom ingress and egress rules {#custom-rules}
+Control inbound and outbound traffic with specific ports and IP addresses for vCluster control plane and workloads:
```

Comment:
Could you add a section showing the actual NetworkPolicy resources that get created when this is enabled? Users would benefit from seeing the default YAML (like `vc-work-{name}` and `vc-cp-{name}`) and knowing that these are managed by vCluster and manual changes on the host cluster will be overwritten.